pkg/controllers/support.go

```diff
@@ -20,7 +20,7 @@ func serviceNameFromCluster(cluster *rayv1.RayCluster) string {
 
 func desiredRayClientRoute(cluster *rayv1.RayCluster) *routeapply.RouteApplyConfiguration {
 	return routeapply.Route(rayClientNameFromCluster(cluster), cluster.Namespace).
-		WithLabels(map[string]string{"ray.io/cluster-name": cluster.Name}).
+		WithLabels(map[string]string{"ray.openshift.ai/cluster-name": cluster.Name}).
```

Comment:
Should that be `RayClusterNameLabel`?

Reply:
Yep, did a find and replace in the controller code. Missed this one. Thanks for catching

Reply:
I also added WithController to the OwnerReferences in support.go

Reply:
For the record, this broke the AppWrapper controller.   I submitted a fix here #640